packages/web-components/src/components/pictogram-item/__stories__/pictogram-item.stories.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import '../pictogram-item';
+import '../../link-with-icon/link-with-icon';
+import ArrowRight20 from 'carbon-web-components/es/icons/arrow--right/20';
+
+import readme from './README.stories.mdx';
+
+export const Default = () => html`
+  <dds-pictogram-item>
+    <svg
+      slot="pictogram"
+      focusable="false"
+      preserveAspectRatio="xMidYMid meet"
+      xmlns="http://www.w3.org/2000/svg"
+      stroke="currentColor"
+      data-autoid="dds--pictogram-item__pictogram"
+      aria-label="Pictogram description"
+      width="48"
+      height="48"
+      viewBox="0 0 48 48"
+      role="img"
+      class="bx--pictogram-item__pictogram"
+    >
+      <path
+        fill="none"
+        stroke-linejoin="round"
+        stroke-miterlimit="10"
+        stroke-width=".72"
+        d="M37,32 H11c-1.1,0-2-0.9-2-2V13c0-1.1,0.9-2,2-2h26c1.1,
+        0,2,0.9,2,2v17C39,31.1,38.1,32,37,32z M17,37h14 M24,32v5 M9,27h30"
+      ></path>
+    </svg>
+    <span slot="heading">Lorem ipsum dolor sit</span>
+    <span slot="copy">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
@RaphaelAmadeu can you make these into knobs?

Reply:
@jeffchew This was added on my latest commit